pkg/helper/helper.go

```diff
@@ -0,0 +1,260 @@
+package helper
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/openshift-pipelines/release-tests/pkg/client"
+	"github.com/openshift-pipelines/release-tests/pkg/config"
+	secv1 "github.com/openshift/api/security/v1"
+	secclient "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
+	op "github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/names"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	knativetest "knative.dev/pkg/test"
+)
+
+// AssertNoError confirms the error returned is nil
+// func AssertNoError(t *testing.T, err error) {
+// 	t.Helper()
+// 	if err != nil {
+// 		t.Fatal(err)
+// 	}
+// }
+
+// AssertNoError confirms the error returned is null
+func AssertNoError(err error, description string) {
+	Expect(err).ShouldNot(HaveOccurred(), description)
+}
+
+// NewClientSet is a setup function which helps you
+// 	1. to creates clientSet instance to `client`
+// 	2. creates Random namspace
+func NewClientSet() (*client.Clients, string, func()) {
+	ns := names.SimpleNameGenerator.RestrictLengthWithRandomSuffix("testrelease")
+	cs := client.NewClients(knativetest.Flags.Kubeconfig, knativetest.Flags.Cluster, ns)
+	CreateNamespace(cs.KubeClient, ns)
```

Comment:
I noticed that it didn't create a namespace on my cluster and it didn't log any error.

Reply:
It will create a namespace, and gets deleted once you complete execution

Reply:
I was watching the cluster **during** test execution and didn't see it there.